src/Stratis.Bitcoin.Features.MemoryPool/Rules/CheckTxOutDustRule.cs

```diff
@@ -22,10 +22,15 @@ public sealed class CheckTxOutDustRule : MempoolRule
         /// <inheritdoc />
         public override void CheckTransaction(MempoolValidationContext context)
         {
+            // If there are more inputs than outputs we don't care if they are dust transactions
+            // as we are effectively consolidating utxo's on the network.
+            if (context.Transaction.Inputs.Count > context.Transaction.Outputs.Count)
```

Comment:
I am not sure this is a good change.

Reply:
So to attack the network I create many small outputs that are just above dust, and then I cut that number by half to get two inputs and one dust output and the relay will allow this to propagate.

Reply:
I've done some analysis and I agree with @dangershony. Let's remove that change which I've proposed earlier.